source/development_process/2_deployment.rst

```diff
@@ -283,17 +285,24 @@ How to Write a Light Weight Resource Provider
 ---------------------------------------------
 
 A Light Weight Resource Provider, or LWRP, is a simple way to write custom
-reusable components for configuration. For instance, one could copy a sysV init
-file for apache into ``/etc/init.d`` in every cookbook which needs to setup
-apache, or one could provide a resource which will set the status of the init
-script, whether or not it is enabled etc., and automatically copies it over
-for you.
+reusable components for configuration. They can be used to do things like:
+
+- Automating common tasks in your chef cookbooks
```

Comment:
Stylistically we use `*` to enumerate items in a list. RST doesn't care, but please use `*` for consistency.